Author: Daniel Baumann <mail@daniel-baumann.ch>
Description: Ignoring udev seat tags for the time being (Closes: #751726).

--- a/src/libply-splash-core/ply-device-manager.c
+++ b/src/libply-splash-core/ply-device-manager.c
@@ -329,7 +329,7 @@ create_seats_for_subsystem (ply_device_m
                         /* We only care about devices assigned to a (any) seat. Floating
                          * devices should be ignored.
                          */
-                        if (udev_device_has_tag (device, "seat")) {
+                        if (true) {
                                 const char *node;
                                 node = udev_device_get_devnode (device);
                                 if (node != NULL) {
